consensus/src/epoch_manager.rs

```diff
@@ -542,6 +534,35 @@ impl EpochManager {
         self.block_retrieval_tx = None;
     }
 
+    async fn start_recovery_manager(
+        &mut self,
+        ledger_data: LedgerRecoveryData,
+        epoch_state: EpochState,
+    ) {
+        let network_sender = NetworkSender::new(
+            self.author,
+            self.network_sender.clone(),
+            self.self_sender.clone(),
+            epoch_state.verifier.clone(),
+        );
+        let (round_manager_tx, round_manager_rx) = aptos_channel::new(
+            QueueStyle::LIFO,
+            1,
+            Some(&counters::ROUND_MANAGER_CHANNEL_MSGS),
+        );
+        self.round_manager_tx = Some(round_manager_tx);
```

Comment:
Let's rename this to `round_or_recovery_manager_tx` or something like that to avoid confusion.